### PR TITLE
serving index of groups

### PR DESCRIPTION
**Pull request by lukeaiken**

### CODE REVIEW

File: db/seeds.rb [hunk 1]
```diff
@@ -3,11 +3,16 @@
...
 # Examples:
 #
-#   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
-#   Mayor.create(name: 'Emanuel', city: cities.first)
-Skill.create([{ name: 'skill1', featured: true },
-              { name: 'skill2', featured: true },
-              { name: 'skill3', featured: false },
-              { name: 'skill4', featured: true },
-              { name: 'skill5', featured: false },
+#   cities = City.create([{ name: "Chicago" }, { name: "Copenhagen" }])
```
Comment (lukeaiken) [inline db/seeds.rb]: what do you think about removing these comments?

File: db/seeds.rb [hunk 2]
```diff
@@ -3,11 +3,16 @@
...
+#   Mayor.create(name: "Emanuel", city: cities.first)
```
Comment (lukeaiken) [inline db/seeds.rb]: How would you feel about using `factory_girl_rails` to create seeds? :spaghetti:

File: spec/controllers/api/v1/group_controller_spec.rb [hunk 3]
```diff
@@ -0,0 +1,22 @@
+describe Api::V1::GroupsController, type: :controller do
```
Comment (lukeaiken) [inline spec/controllers/api/v1/group_controller_spec.rb]: please remove `type: :controller`

File: spec/controllers/api/v1/group_controller_spec.rb [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+
+  describe "index" do
+
+    before do
```
Comment (lukeaiken) [inline spec/controllers/api/v1/group_controller_spec.rb]: please do not use `before` blocks

File: spec/controllers/api/v1/group_controller_spec.rb [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+      2.times { create(:group) }
+    end
+
+    it "returns a json array of all groups" do
+      get :index, format: :json
+
+      expect(response.status).to eq 200
+      expect(json_last_group["name"]).to eq("group_2")
```
Comment (lukeaiken) [inline spec/controllers/api/v1/group_controller_spec.rb]: is this a mystery guest?

File: spec/controllers/api/v1/group_controller_spec.rb [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+    end
+  end
+
+  private
+
+  def json_last_group
+    parsed_json_response_body["group"].last
```
Comment (a6ftcruton) [inline spec/controllers/api/v1/group_controller_spec.rb:19]: How you feel about not inviting mystery guests to your dinner party? :dancer:

Comment (lukeaiken) [inline spec/controllers/api/v1/group_controller_spec.rb:19]: I believe this is resolved.

**Pull request merged**